### PR TITLE
[Notifier] [TurboSms] Better tests

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | Follows #58242
| License       | MIT

Still waiting for #58242 to get unmerged, to fix also the newly introduced test.

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/TurboSms/Tests/TurboSmsTransportTest.php [hunk 1]
```diff
@@ -128,10 +109,8 @@ public function testInvalidFrom()
         $this->expectException(LengthException::class);
         $this->expectExceptionMessage('The sender length of a TurboSMS message must not exceed 20 characters.');
 
-        $message = new SmsMessage('380931234567', 'Hello!');
-        $transport = new TurboSmsTransport('authToken', 'abcdefghijklmnopqrstu', $this->createMock(HttpClientInterface::class));
+        new TurboSmsTransport('authToken', 'abcdefghijklmnopqrstu', $this->createMock(HttpClientInterface::class));
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/TurboSms/Tests/TurboSmsTransportTest.php:139]: the exception happens already in `TurboSmsTransport::constructor()`, no need to call `send()`

**Comment by OskarStark** [comment]
Rebased and ready to merge from my side

**Review by alexandre-daubois — APPROVED** [review]

File: src/Symfony/Component/Notifier/Bridge/TurboSms/Tests/TurboSmsTransportTest.php [hunk 2]
```diff
@@ -165,10 +136,8 @@ public function testInvalidFrom()
...
```
Comment (xabbuh) [inline src/Symfony/Component/Notifier/Bridge/TurboSms/Tests/TurboSmsTransportTest.php]: this blank line should be removed

Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/TurboSms/Tests/TurboSmsTransportTest.php]:
```suggestion
```

**Review by xabbuh — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @OskarStark.

**Pull request merged**